platform/javascript/os_javascript.cpp

```diff
@@ -687,7 +687,7 @@ OS::VideoMode OS_JavaScript::get_video_mode(int p_screen) const {
 
 Size2 OS_JavaScript::get_screen_size(int p_screen) const {
 
-	ERR_FAIL_COND_V(p_screen != 0, Size2());
+	//ERR_FAIL_COND_V(p_screen != 0, Size2());
```

Comment:
I guess you could remove that line completely, it didn't really make sense previously to limit to 0 anyway.